config/config-sandbox.yaml

```diff
@@ -0,0 +1,19 @@
+admin:
+  # For GRPC endpoints you might want to use dns:///flyte.myexample.com
+  endpoint: localhost:30081
+  authType: Pkce
+  insecure: true
+logger:
+  show-source: true
+  level: 0
+storage:
+  connection:
+    access-key: minio
+    auth-type: accesskey
+    disable-ssl: true
+    endpoint: http://localhost:30084
+    region: us-east-1
+    secret-key: miniostorage
+  type: minio
+  container: "my-s3-bucket"
+  enable-multicontainer: true
```

Comment:
we shouldn't need this as of https://github.com/flyteorg/flytekit/pull/907 (just remove `storage`).